### DISCUSSION TITLE
Custom events

### DISCUSSION THREAD

**Discussion by NoWasteDev** [pixijs/pixijs#9311] [Q&A]
There used to be a way to communicate between two components via custom events + payload.
Now I'm getting an error message:
Argument of type '"<customEvent>"' is not assignable to parameter of type 'keyof DisplayObjectEvents'.ts(2345)

How do I notify components of custom events?

**Comment by SuperSodaSea** [comment]
:arrow_right: Issue #8957

**Reply by NoWasteDev** [reply]
@SuperSodaSea thank you for the super speedy reply. I'm aware of this, but it's also clear that the workaround is far from complete. Is there a recommended strategy for now? Clearly component communication is the barebones essence of game development.
